packages/nc-gui/components/erd/TableNode.vue

```diff
@@ -54,7 +54,7 @@ watch(
     </template>
 
     <div
-      class="relative h-full flex flex-col justify-center bg-white min-w-16 min-h-8 rounded-lg nc-erd-table-node"
+      class="relative h-full max-w-[20rem] flex flex-col justify-center bg-white min-w-16 min-h-8 rounded-lg nc-erd-table-node"
```

Comment:
Better to use standard tailwind class for`max-w-[20rem]` which would be `w-80`

Reply:
**fixed in below Commit**

[fix : removed unwanted ! - important](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)
[b862466](https://github.com/nocodb/nocodb/pull/7114/commits/b862466785ff4be99629740e90baa00bb3b14731)